src/main.rs

```diff
@@ -51,17 +51,15 @@
 //! cargo install repgrep
 //! ```
 //!
-//! ### From the AUR (Arch Linux)
```

Comment:
Thanks for updating both the doc comments and the README file :heart: 

It's not clear since I haven't documented it anywhere, but `README.md` is generated by the doc comments using `cargo readme`. And that tool indents all the markdown headings by one `#`.

So, if you change this to be `###` like the other headings around it in `src/main.rs`, and then run `just readme` it will generate `README.md` with the right formatting.

---

I'm happy to do this for you if you don't want to `cargo install cargo-readme`, just let me know.

Reply:
Oh, didn't realize that. It is a pretty cool automation!

Feel free to go ahead and update it 🐻